Resources/Prototypes/Loadouts/Jobs/Engineering/station_engineer.yml

```diff
@@ -2,23 +2,12 @@
 - type: loadoutEffectGroup
   id: SeniorEngineering
   effects:
-  - !type:JobRequirementLoadoutEffect
```

Comment:
If these are Wizden requirements then this needs to be commented out instead of being deleted, and needs Delta-V comments